optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/roster/RosterRestService.java

```diff
@@ -97,9 +105,20 @@ EmployeeRosterView getEmployeeRosterViewFor(@ApiParam(required = true) @PathPara
     @Path("/terminate")
     void terminateRosterEarly(@ApiParam(required = true) @PathParam("tenantId") Integer tenantId);
 
+    @ApiOperation("Publishes the next set of draft shifts and creates new draft shift from the rotation template.")
```

Comment:
Use active tense, just like the other docs, so "publish" instead of "Publishes" and "create" instead of "creates".
There's a general docs/javadocs convention to prefer active tense whenever possible and to avoid past or future tense.